src/libraries/Common/src/System/Security/Cryptography/Asn1/Pkcs12/MacData.xml.cs

```diff
@@ -48,7 +48,12 @@ internal void Encode(AsnWriter writer, Asn1Tag tag)
 
             // DEFAULT value handler for IterationCount.
             {
+#if NET7_0_OR_GREATER
+                const int AsnManagedIntegerDerMaxEncodeSize = 6;
+                AsnWriter tmp = new AsnWriter(AsnEncodingRules.DER, initialCapacity: AsnManagedIntegerDerMaxEncodeSize);
```

Comment:
For my education: why do we need `#ifdefs` for this project? I can see that it targets multiple TFMs:

https://github.com/dotnet/runtime/blob/09a3007460bee01bdd90280647d82ff7c2b8c518/src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj#L3

And the ctor that you are using here is not conditionally excluded:

https://github.com/dotnet/runtime/blob/09a3007460bee01bdd90280647d82ff7c2b8c518/src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnWriter.cs#L65

And it's also included in the `ref` assembly:

https://github.com/dotnet/runtime/blob/09a3007460bee01bdd90280647d82ff7c2b8c518/src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.cs#L147

But for some reason https://apisof.net/catalog/ab4b42b3-b2d0-1159-be5b-7bae6dd7a092 reports that it supports only .NET 7.

Reply:
> get rid of the #ifdef.

Pretend I wrote something that makes sense in C#.